### DISCUSSION TITLE
Good first issues / Help wanted minimum info

### DISCUSSION THREAD

**Discussion by edgarfgp** [dotnet/fsharp#13970] [General]
I have started contributing to the compiler about 6 months ago,  and while looking at issues to contribute, I notice that most of them have only the corresponding tags like ```Are-Compiler-Checking```(The tags are really helpful TBH) etc . But sometimes that is not enough for people like me to understand where to start looking :( 

Im not advocating for a long description or possible solution for the bug, Just enough info to new contributors

I'd like to hear from the F# team  : 

- Providing a little bit of a compiler technical description will be ok ?

- Maybe having a "Good first issues"  Community Compiler session where we go through the issues and tag them and provide that info 

Thanks for taking the time 😀

**Comment by vzarytovskii** [comment]
Yeah, I think it is a good idea let's see what we can do @dsyme I guess community sessions are helpful.

In addition to that, folks @T-Gro @0101 @psfinaki @abonie are in Slack (I think) and we'll be trying to answer more questions in #compiler channel.

I wouldn't mind doing some pair programming sessions investigating (and documenting) some things.

Regarding the docs - we had an idea to create a bunch of README.md files in the different compiler directories describing what is in them with some extended examples of what each particular part is doing.

**Comment by dsyme** [comment]
> In addition to that, folks @T-Gro @0101 @psfinaki @abonie are in Slack (I think) and we'll be trying to answer more questions in #compiler channel.

@vzarytovskii It's great to have participation there.  That said, I've always had a bit of a problem with the FSSF use of Slack, because there is no external visibility of the considerable community activity that is going on there, and in the big picture that's a real shame.  I think it was always a structural mistake - an active open source community benefits from being seen, and it's part of how a community grows, and it's essential to be able to link into discussions without needing a login or permissions.

I don't particularly know what to do about this now but I do feel it's unfortunate that we hide activity like this.

**Comment by edgarfgp** [comment]
@dsyme @vzarytovskii Personally the way I can help is by streaming some of my contributions to the compiler and linking to a particular issue/ PR. I'm considering having a regular schedule during my working hours(Need to agree with my employer). Would love to have the F# team's involvement.
